protos/install_grpc_plugins.sh

```diff
@@ -0,0 +1,16 @@
+echo "This script install grpc_python_plugin from source because of this https://github.com/grpc/grpc/issues/15675#issuecomment-397038811"
```

Comment:
Alternative solution:

`yarn global add protoc-gen-grpc-web`

And it's cross-platform.

Reply:
Je peux le add à https://github.com/PolyCortex/polydodo/pull/18

Reply:
Pour python, lire https://github.com/PolyCortex/polydodo/pull/24/files/610d1a40f5a5daa62852fe2e372b6ea327e2aa79#r493842511